spring-cloud-aws-autoconfigure/pom.xml

```diff
@@ -78,19 +67,12 @@
 			<artifactId>javax.mail-api</artifactId>
 			<optional>true</optional>
 		</dependency>
-		<dependency>
-			<groupId>software.amazon.awssdk</groupId>
-			<artifactId>auth</artifactId>
-		</dependency>
 		<dependency>
 			<groupId>software.amazon.awssdk</groupId>
 			<artifactId>ssm</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>software.amazon.awssdk</groupId>
-			<artifactId>s3</artifactId>
```

Comment:
S3 should not be removed.

Reply:
Above we are pulling:

```xml
<dependency>
			<groupId>io.awspring.cloud</groupId>
			<artifactId>spring-cloud-aws-s3</artifactId>
			<optional>true</optional>
		</dependency>
```

So this s3 dependency is redundant.